### DISCUSSION TITLE
Win7 odd startup problem

### DISCUSSION THREAD

**Discussion by dgoadby** [jziolkowski/tdm#234] [Q&A]
I downloaded the latest version (0.2.13) and ran it. It started up and all looked ok until I gave it the incorrect MQTT server address. 

The application crashed with no messages.

Now, if I try to run the application there is a wait followed by a flash on the screen and then it dies. 

I checked the task manager and I don't see any orphan tasks. I'm cannot reboot the PC for a while but any ideas what is going on?

**Comment by barbudor** [comment]
I'm unsure which version of python is used to build the Windows executable, but Windows 7 is no more supported by Python since v3.9
And it is likely that v3.9 or greater is used

You MAY be lucky by running the python code directly with a Python environment in your computer but that's not even guaranteed.
Personnaly I don't have anymore a Win7 machine available for testing.

**Comment by jziolkowski** [comment]
> I downloaded the latest version (0.2.13) and ran it. It started up and all looked ok until I gave it the incorrect MQTT server address. 
> 
> The application crashed with no messages.
> 
> Now, if I try to run the application there is a wait followed by a flash on the screen and then it dies. 
> 
> I checked the task manager and I don't see any orphan tasks. I'm cannot reboot the PC for a while but any ideas what is going on? 

Read the project wiki and find location of the config file. Change the connect on startup flag to false and/or set MQTT broker to proper host. Should fix the problem. I'll have a look for a better error handling in such case.

**Comment by Jason2866** [comment]
Build with Python 3.7 or 3.8 (changed in December). It is for sure not build with Python v3.9

**Reply by barbudor** [reply]
ok, so that's not the problem though
Thanks @Jason2866
